Splunk_TA_paloalto/default/props.conf

```diff
@@ -106,7 +106,7 @@ EVAL-report_id       = if(log_subtype=="wildfire", coalesce(report_id,threat_id)
 EVAL-http_category   = if(log_subtype=="url", raw_category, null())
 EVAL-verdict         = if(log_subtype=="wildfire", raw_category, null())
 EVAL-threat_category = if(log_subtype!="url" AND log_subtype!="file", if(threat_category=="unknown",log_subtype,coalesce(threat_category,log_subtype)), null())
-EVAL-category        = if(log_subtype=="url" OR log_subtype=="file", raw_category, threat_category)
+EVAL-category        = if(log_subtype=="url" OR log_subtype=="file", split(coalesce(new_category, raw_category), ","), threat_category)
```

Comment:
There is no `new_category` field.

Reply:
Based on the changes in transforms.conf `new_category` here looks like it should be `url_category_list`.

Reply:
updated this to be url_category